src/app/course/course.component.html

```diff
@@ -1,43 +1,55 @@
 <section class="tui-container tui-container_fullwidth">
-    <ng-container *ngIf="breadCrumbs && course else skeletonBreadcrumbs">
-        <tui-breadcrumbs class="my-3" [size]="'l'">
-            <ng-container *ngFor="let item of breadCrumbs">
-                <a
-                    *tuiBreadcrumb
-                    tuiLink
-                    [routerLink]="item.routerLink">
-                    {{item.caption}}
-                </a>
-            </ng-container>
-        </tui-breadcrumbs>
-    </ng-container>
-    <ng-template #skeletonBreadcrumbs>
-        <tui-breadcrumbs class="my-3" [size]="'l'">
-            <a *tuiBreadcrumb tuiLink class="tui-skeleton">crumb that is longer</a>
-            <a *tuiBreadcrumb tuiLink class="tui-skeleton">crumb that is longer</a>
-            <a *tuiBreadcrumb tuiLink class="tui-skeleton">crumb that is longer</a>
-        </tui-breadcrumbs>
-    </ng-template>
+    <div class="top-box my-3">
+        <ng-container *ngIf="breadCrumbs && course else skeletonBreadcrumbs">
+            <tui-breadcrumbs [size]="'l'">
+                <ng-container *ngFor="let item of breadCrumbs">
+                    <a
+                        *tuiBreadcrumb
+                        tuiLink
+                        [routerLink]="item.routerLink">
+                        {{item.caption}}
+                    </a>
+                </ng-container>
+            </tui-breadcrumbs>
+        </ng-container>
+        <ng-template #skeletonBreadcrumbs>
+            <tui-breadcrumbs class="my-3" [size]="'l'">
+                <a *tuiBreadcrumb tuiLink class="tui-skeleton">crumb that is longer</a>
+                <a *tuiBreadcrumb tuiLink class="tui-skeleton">crumb that is longer</a>
+                <a *tuiBreadcrumb tuiLink class="tui-skeleton">crumb that is longer</a>
+            </tui-breadcrumbs>
+        </ng-template>
+        <a
+            *ngIf="user?.is_teacher && onHomepage"
```

Comment:
Wrong permissions. use `course.has_edit_permission`. Also, let's have it everywhere.